bazel/tools/java/com/code_intelligence/jazzer/tools/FuzzTargetTestWrapper.java

```diff
@@ -92,13 +96,13 @@ public static void main(String[] args) {
         System.exit(4);
       }
       // Verify that libFuzzer dumped a crashing input.
-      if (verifyCrashInput
+      if (JAZZER_CI && verifyCrashInput
```

Comment:
Why should this only be verified on CI? Also checks below.

Reply:
So the assumption is that someone executing tests is primarily interested in reproducing the findings and verifying the reproducers is a CI or regression topic. In that case let's leave the name as is.